homeassistant/components/homewizard/config_flow.py

```diff
@@ -305,65 +305,65 @@ async def async_step_reconfigure(
         )
 
-    @staticmethod
-    async def _async_try_connect(ip_address: str) -> Device:
-        """Try to connect.
 
-        Make connection with device to test the connection
-        and to get info for unique_id.
-        """
+async def async_try_connect(ip_address: str) -> Device:
+    """Try to connect.
 
-        energy_api: HomeWizardEnergy
+    Make connection with device to test the connection
+    and to get info for unique_id.
+    """
 
-        # Determine if device is v1 or v2 capable
-        if await has_v2_api(ip_address):
-            energy_api = HomeWizardEnergyV2(ip_address)
-        else:
-            energy_api = HomeWizardEnergyV1(ip_address)
+    energy_api: HomeWizardEnergy
 
-        try:
-            return await energy_api.device()
+    # Determine if device is v1 or v2 capable
+    if await has_v2_api(ip_address):
+        energy_api = HomeWizardEnergyV2(ip_address)
+    else:
+        energy_api = HomeWizardEnergyV1(ip_address)
 
-        except DisabledError as ex:
-            raise RecoverableError(
-                "API disabled, API must be enabled in the app", "api_not_enabled"
-            ) from ex
+    try:
+        return await energy_api.device()
 
-        except UnsupportedError as ex:
-            LOGGER.error("API version unsuppored")
-            raise AbortFlow("unsupported_api_version") from ex
+    except DisabledError as ex:
+        raise RecoverableError(
+            "API disabled, API must be enabled in the app", "api_not_enabled"
+        ) from ex
 
-        except RequestError as ex:
-            raise RecoverableError(
-                "Device unreachable or unexpected response", "network_error"
-            ) from ex
+    except UnsupportedError as ex:
+        LOGGER.error("API version unsuppored")
+        raise AbortFlow("unsupported_api_version") from ex
 
-        except UnauthorizedError as ex:
-            raise UnauthorizedError("Unauthorized") from ex
+    except RequestError as ex:
+        raise RecoverableError(
+            "Device unreachable or unexpected response", "network_error"
+        ) from ex
 
-        except Exception as ex:
-            LOGGER.exception("Unexpected exception")
-            raise AbortFlow("unknown_error") from ex
+    except UnauthorizedError as ex:
+        raise UnauthorizedError("Unauthorized") from ex
 
-        finally:
-            await energy_api.close()
+    except Exception as ex:
+        LOGGER.exception("Unexpected exception")
+        raise AbortFlow("unknown_error") from ex
 
-    @staticmethod
-    async def _async_request_token(ip_address: str) -> str | None:
-        """Try to request a token from the device.
+    finally:
+        await energy_api.close()
 
-        This method is used to request a token from the device,
-        it will return None if the token request failed.
-        """
 
-        api = HomeWizardEnergyV2(ip_address)
+async def async_request_token(ip_address: str) -> str | None:
```

Comment:
Moved this from `HomeWizardConfigFlow._async_request_token` to make it accessible by repairs.py. No code changes.